components/post_view/post_header/post_header.tsx

```diff
@@ -151,6 +151,7 @@ export default class PostHeader extends React.PureComponent<Props> {
         } else if (isSystemMessage) {
             userProfile = (
                 <UserProfile
+                    userId=''
```

Comment:
I think it makes sense to be explicit here by making this required. I'm surprised this was optional to begin with. :+1: